modules/playlists/src/main/java/org/opencastproject/playlists/PlaylistAccessControlEntry.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+package org.opencastproject.playlists;
+
+import org.opencastproject.security.api.AccessControlEntry;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+/**
+ * This has the same fields as an {@link AccessControlEntry}, but since playlists purely exist in the database, ACLs
+ * need to be persisted alongside them, which is what this is for.
+ */
+@Entity(name = "PlaylistAccessControlEntry")
+@Table(name = "oc_playlist_access_control_entry")
+public class PlaylistAccessControlEntry {
+  @Id
+  @GeneratedValue
+  @Column(name = "id")
+  private long id;
+
+  @Column(name = "allow")
+  private boolean allow;
+
+  @Column(name = "role")
+  private String role;
+
+  @Column(name = "action")
+  private String action;
```

Comment:
Although it does not necessarily make much sense, iirc these are are allowed to be null by ACL standards? Would have to look that up.

Reply:
I suppose we can just decide that for our playlists they cannot be null. It would just lead to random errors down the lines I suppose. We all know those "this value should never be X, but somehow it is" errors with Opencast :D 

But I won't block this PR on this.